src/Codeception/Util/Shared/Asserts.php

```diff
@@ -189,466 +94,27 @@ protected function assertNotRegExp($pattern, $string, $message = '')
-    }
-
-    /**
-     * @param $expected
-     * @param $actual
-     * @param $description
-     */
-    protected function assertGreaterOrEquals($expected, $actual, $description = '')
-    {
-        Assert::assertGreaterThanOrEqual($expected, $actual, $description);
-    }
-
-    /**
-     * @param $expected
-     * @param $actual
-     * @param $description
-     */
-    protected function assertLessOrEquals($expected, $actual, $description = '')
-    {
-        Assert::assertLessThanOrEqual($expected, $actual, $description);
-    }
-
-    /**
-     * @param $actual
-     * @param $description
-     */
-    protected function assertIsEmpty($actual, $description = '')
-    {
-        Assert::assertEmpty($actual, $description);
-    }
-
-    /**
-     * @param $key
-     * @param $actual
-     * @param $description
-     */
-    protected function assertArrayHasKey($key, $actual, $description = '')
-    {
-        Assert::assertArrayHasKey($key, $actual, $description);
-    }
-
-    /**
-     * @param $key
-     * @param $actual
-     * @param $description
-     */
-    protected function assertArrayNotHasKey($key, $actual, $description = '')
-    {
-        Assert::assertArrayNotHasKey($key, $actual, $description);
-    }
-
-    /**
-     * Checks that array contains subset.
-     *
-     * @param array  $subset
-     * @param array  $array
-     * @param bool   $strict
-     * @param string $message
-     */
-    protected function assertArraySubset($subset, $array, $strict = false, $message = '')
-    {
-        Assert::assertArraySubset($subset, $array, $strict, $message);
-    }
-
-    /**
-     * @param $expectedCount
-     * @param $actual
-     * @param $description
-     */
-    protected function assertCount($expectedCount, $actual, $description = '')
-    {
-        Assert::assertCount($expectedCount, $actual, $description);
-    }
-
-    /**
-     * @param $class
-     * @param $actual
-     * @param $description
-     */
-    protected function assertInstanceOf($class, $actual, $description = '')
-    {
-        Assert::assertInstanceOf($class, $actual, $description);
-    }
-
-    /**
-     * @param $class
-     * @param $actual
-     * @param $description
-     */
-    protected function assertNotInstanceOf($class, $actual, $description = '')
-    {
-        Assert::assertNotInstanceOf($class, $actual, $description);
-    }
-
-    /**
-     * @param $type
-     * @param $actual
-     * @param $description
-     */
-    protected function assertInternalType($type, $actual, $description = '')
-    {
-        Assert::assertInternalType($type, $actual, $description);
-    }
-    
-    /**
-     * Fails the test with message.
-     *
-     * @param $message
-     */
-    protected function fail($message)
-    {
-        Assert::fail($message);
-    }
-
-    protected function assertStringContainsString($needle, $haystack, $message = '')
-    {
-        TestCase::assertStringContainsString($needle, $haystack, $message);
```

Comment:
Right, you removed my wrapper methods here, so they are broken with older versions of PHPUnit.

Reply:
Sorry, It wasn't my intention, but I'm glad it could be fixed, thanks you